extra-multimedia/feeluown-xiami/autobuild/defines

```diff
@@ -0,0 +1,10 @@
+PKGNAME="feeluown-xiami"
+PKGSEC=sound
+PKGDES="a FeelUOwn plugin for Xiami"
```

Comment:
First letter in `PKGDES=` should be capitalised; the "A" is actually not necessary here.